src/views/visual/visual-view-graph.styles.js

```diff
@@ -0,0 +1,30 @@
+import { css } from 'lit-element';
+
+export default css`
+  #parallels-chart {
+    font: 13px sans-serif;
+    padding: 30px;
+    width: 93vw;
```

Comment:
if possible, it's better to use percentages here, as we might want to put something else on the page later, which would break these styles.